### PR TITLE
Improved the cache flush command

### PR DESCRIPTION
**Pull request by Nyholm**
You can now clear session, router and doctrine separately.

### CODE REVIEW

File: src/Command/CacheFlushCommand.php [hunk 1]
```diff
@@ -32,7 +33,7 @@ protected function configure()
     {
         $this->setName('cache:flush');
         $this->setDescription('Flushes the given cache');
-        $this->addArgument('instance', InputArgument::REQUIRED, 'Which instance do you want to clean?');
+        $this->addArgument('type', InputArgument::REQUIRED, 'Which type of cache do you want to clear?');
```
Comment (cryptiklemur) [inline src/Command/CacheFlushCommand.php:35]: Should the user be able to clear all of them?

Comment (Nyholm) [inline src/Command/CacheFlushCommand.php:35]: Yeah, Im was thinking of a "all" option. I should add that.

**Comment by Nyholm** [comment]
Im doing checks to see if the cache implements TaggablePoolInterface. Do we need to require that in composer.json?

**Comment by cryptiklemur** [comment]
uh. I'm not sure... I think so though

**Comment by Nyholm** [comment]
Updated !

File: src/Command/CacheFlushCommand.php (hunk 1 above)
Comment (cryptiklemur) [inline src/Command/CacheFlushCommand.php:35]: yeah, probably

Comment (cryptiklemur) [inline src/Command/CacheFlushCommand.php:35]: which you did :+1:

**Comment by Nyholm** [comment]
Thank you!

**Pull request merged**